esmvaltool/diag_scripts/shared/plot/styles_python/style_header

```diff
@@ -1,32 +1,33 @@
 ###############################################################################
 # PYTHON STYLE FILE
-# created by 'convert_ncl_style.py'
+# {0}
```

Comment:
instead of 0 and 1 etc, use e.g. `{output_file}` and format with `some_string.format(output_file=OUTPUT_FILE)`